Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:14.04
+RUN apt-get update && apt-get install -y \
+    python-qt4 \
+    python-pyside \
+    python3-pyqt4 \
```

Comment:
We can probably strip the Python 3 libraries, as we're only running the tests with Python 2 anyway, and it's probably unnecessary to run both locally anyway.